nix/devshell/kernel.nix

```diff
@@ -44,7 +44,7 @@
           export PS1="[ghaf-kernel-${platform}-devshell:\w]$ "
         '';
         # use "eval $checkPhase" - see https://discourse.nixos.org/t/nix-develop-and-checkphase/25707
-        checkPhase = "cp ../modules/host/ghaf_host_hardened_baseline-${platform} ./.config && make -j$(nproc)";
+        checkPhase = "cp ../modules/hardware/x86_64-generic/kernel/configs/ghaf_host_hardened_baseline-${platform} ./.config && make -j$(nproc)";
```

Comment:
Using of `${platform}` is wrong here, `x86_64-generic` already a platform, so here should be something like `../modules/${platform}/kernel/configs/ghaf_baseline`

Reply:
I have addressed review comment.